src/open-uri.c

```diff
@@ -925,6 +925,31 @@ handle_open_in_thread_func (GTask *task,
                                                      g_object_ref (request));
 }
 
+static gboolean
+handle_scheme_supported (XdpDbusOpenURI *object,
+                         GDBusMethodInvocation *invocation,
+                         const gchar *arg_scheme,
+                         GVariant *arg_options)
+{
+  g_autoptr(GAppInfo) app_info = NULL;
+
```

Comment:
I think if we add options, we should check them and filter out unsupported options (ie all of them, currently)

Reply:
Apart from that, +1 from me.